content/guides/getting-started/creating-a-cy-mount-command.md

```diff
@@ -2,4 +2,143 @@
 title: Creating a cy.mount Command
 ---
 
-CONTENT TBD
+We recommend creating a custom `cy.mount` command which wraps the mount command
+from the framework-specific libraries in your component tests. Doing so offers a
+few advantages:
+
+- You don't need to import the mount command into every test as the `cy.mount`
+  command is available globally
+- You can set up common scenarios that you usually have to do in each test, like
+  wrapping a React component in a provider or adding Vue plugins
+
+Cypress does not provide a `cy.mount` command out-of-the-box. If you attempt to
+use `cy.mount` before creating it, you will get a warning letting you know:
+
+<img src="/_nuxt/assets/img/guides/component-testing/cy-mount-must-be-implemented.png" alt="cy.mount must be implemented by the user." />
+
+Let's take a look at how to implement one.
+
+## Creating a New `cy.mount` Command
+
+To use `cy.mount` you will need to add a custom command to the commands file.
+Below are examples that you can start with for your commands:
+
+:::react-vue-example
+
+```js
+import { mount } from '@cypress/react'
+Cypress.Commands.add('mount', (jsx, options) => {
+  // Wrap any parent components needed
+  // ie: return mount(<MyProvider>{jsx}</MyProvider>, options)
+  return mount(jsx, options)
+})
+```
+
+```js
+import { mount } from '@cypress/vue'
```

Comment:
Like with the react example, I'd like to see an examples using one or two common Vue plugins to really show people what this could look like